readme.md

```diff
@@ -385,31 +432,34 @@ test(async t => {
 });
 ```
 
-*You don't have to manually call `t.end()`.*
-
 ### Observable support
 
 AVA comes with builtin support for [observables](https://github.com/zenparsing/es-observable).
+If you return an observable from a test, AVA will automatically consume it to completion before ending the test.
+*You don't have to use legacy `test.async` mode or `t.end()`.*
 
 ```js
 test(t => {
-	return Observable.of(1, 2, 3).map(n => {
-		t.true(n > 0);
-		return n * n;
-	});
+	t.plan(3);
+	return Observable.of(1, 2, 3, 4, 5, 6)
+		.filter(n => {
+			// only even numbers
+			return n % 2 === 0;
+		})
+		.map(n => {
+			t.pass();
```

Comment:
The previous Observable example was so trivial that I don't feel it adequately explained the advantage AVA offered.
I think this:

 1. Is a simple enough usage of observable that you don't necessarily need to be familiar with them to understand what is happening.
 2. Shows a novel use of `t.plan()` in coordination with the Observable filter. It's a little closer to a test someone might actually right.

@BarryThePenguin 
What I know about Observables I learned in the last 20 minutes, so feel free to tell me where I missed the mark.